### DISCUSSION TITLE
Browser SVG API

### DISCUSSION THREAD

**Discussion by ripter** [skipzero/svg-animation-tool#2] [Q&A]
## `<animate />`

[Spec](https://svgwg.org/specs/animations/#InterfaceSVGAnimationElement)
* Play animation
  * `.beginElement()`
  * `.beginElementAt()`
* Stop animation
  * `.endElement()`
  * `.endElementAt()`

**Comment by skipzero** [comment]
so this is the first action(s) we'll have?
would startElement, startElementAt make more sense? So you'd have a starting El and an end(ing) el. I'm assuming it relates to the element the animation would start at/on and end at/on?

**Reply by ripter** [reply]
> I'm assuming it relates to the element the animation would start at/on and end at/on?

That would make a lot more sense than the actual spec. The value passed into the two *At* versions is an **offset** until it should perform that action.

So `elmAnim.beginElementAt(1)` actually means "begin animation of element in 1 second"
And `elmAnim.endElementAt(5)` means "stop animation of element in 5 seconds"

I still have no idea how to start/stop an animation from a specific point in time.
